src/Form/Type/UiElement/JobOfferUiElementType.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz's SyliusUiElementsPlugin for Sylius.
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusUiElementsPlugin\Form\Type\UiElement;
+
+use MonsieurBiz\SyliusRichEditorPlugin\Attribute\AsUiElement;
+use MonsieurBiz\SyliusRichEditorPlugin\Attribute\TemplatesUiElement;
+use MonsieurBiz\SyliusRichEditorPlugin\Form\Type\WysiwygType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\Extension\Core\Type\UrlType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[AsUiElement(
+    code: 'monsieurbiz_ui_elements.job_offer_ui_element',
+    icon: 'clipboard',
+    title: 'monsieurbiz_ui_elements.ui_element.job_offer_ui_element.title',
+    description: 'monsieurbiz_ui_elements.ui_element.job_offer_ui_element.description',
+    templates: new TemplatesUiElement(
+        adminRender: '@MonsieurBizSyliusUiElementsPlugin/Admin/UiElement/job_offer_ui_element.html.twig',
+        frontRender: '@MonsieurBizSyliusUiElementsPlugin/Shop/UiElement/job_offer_ui_element.html.twig',
+    ),
+    wireframe: '',
```

Comment:
```suggestion
    wireframe: 'job_offer',
```

Can you add the job offer wireframe ?
You can use a blank one like the others, they will be changed soon

![image](https://github.com/user-attachments/assets/1454b4a8-0a4c-42a6-93ba-e5d3208b48dc)

The manifest will copy it on setup : 
https://github.com/monsieurbiz/symfony-recipes/blob/master/monsieurbiz/sylius-ui-elements-plugin/0.4/manifest.json#L11

Reply:
ok